Assets/HoloToolkit/Input/Scripts/InputSources/EditorHandsInput.cs

```diff
@@ -38,47 +41,54 @@ public EditorHandData(uint handId)
             public bool IsFingerDownPending;
             public bool FingerStateChanged;
             public float FingerStateUpdateTimer;
-			public float FingerDownStartTime;
+            public float FingerDownStartTime;
+            public bool ManipulationInProgress;
+            public bool HoldInProgress;
+            public Vector3 CumulativeDelta;
         }
 
         private ManualHandControl manualHandControl;
 
         /// <summary>
-        /// Dispatched each frame that a hand is moving
+        /// Dispatched each frame that a hand is moving.
         /// </summary>
         public event Action<IInputSource, uint> HandMoved;
 
         /// <summary>
         /// Delay before a finger pressed is considered.
         /// This mitigates fake finger taps that can sometimes be detected while the hand is moving.
         /// </summary>
-        private const float FingerPressDelay = 0.07f;
+        private const float fingerPressDelay = 0.07f;
```

Comment:
These constants names should start with an uppercase letter, as was the case originally. Relatively sure that the coding standard across HoloToolkit is to have constants start with an uppercase, even if they are private.